assets/src/blocks/amp-story/edit.js

```diff
@@ -186,6 +208,31 @@ class EditPage extends Component {
 							) }
 						</Fragment>
 					</PanelBody>
+					<PanelBody title={ __( 'Page Settings', 'amp' ) }>
+						<SelectControl
+							label={ __( 'Advance to next page', 'amp' ) }
+							help={ 'media' === autoAdvanceAfter ? __( 'Based on the first video encountered on the page', 'amp' ) : undefined }
+							value={ autoAdvanceAfter }
+							options={ autoAdvanceAfterOptions }
+							onChange={ ( value ) => {
+								setAttributes( { autoAdvanceAfter: value } );
+
+								if ( 'media' === value ) {
+									setAttributes( { autoAdvanceAfterMedia: firstMediaBlock.attributes.anchor } );
+								}
+							} }
+						/>
+						{ 'time' === autoAdvanceAfter && (
+							<RangeControl
+								label={ __( 'Time in seconds', 'amp' ) }
+								value={ autoAdvanceAfterDuration ? parseInt( autoAdvanceAfterDuration ) : 0 }
+								onChange={ ( value ) => setAttributes( { autoAdvanceAfterDuration: value } ) }
+								min={ totalAnimationDuration }
+								initialPosition={ totalAnimationDuration }
+								help={ totalAnimationDuration > 1 ? __( 'A minimum time is enforced because there are animated blocks on this page', 'amp' ) : undefined }
```

Comment:
For some reason enforcing a minimum time doesn't seem to work for me.
Created a new AMP Story, added Text block and Image block, both with animations, without delay and both starting Immediately, then went to Page options and chose "After a certain time", it still displayed 0. Could you confirm that it's working as expected for you?

Reply:
Not at the computer anymore, but will check this again.

Reply:
I think I just fixed this with f25b433ad43b094f732c122d96d943e35d7fee9a as well.